src/components/forms/ProjectGrantsForm.tsx

```diff
@@ -1300,13 +1301,21 @@ export const ProjectGrantsForm: FC = () => {
                       selectedOptionColor='brand.option'
                       chakraStyles={chakraStyles}
                     />
+
+                    {error && (
+                      <Box mt={1}>
+                        <PageText as='small' fontSize='helpText' color='red.500'>
+                          Country is required.
+                        </PageText>
+                      </Box>
+                    )}
```

Comment:
@pettinarip Noting that this will never show up... the selector defaults to empty, but it's impossible to edit this field to trigger the error without selecting a valid option. To trigger the error, this field needs to change, and there is no way to select an empty string.
Should this trigger on _blur_, so if the user tabs over the field it will show the error after it had focus? The red asterisk is there to signify that it's required, but if a user skips this, the form will not validate and there won't be any guidance as to specifically why.

Reply:
Nice catch @wackerow! and agreed that we should check this on blur! adding a commit soon